client/my-sites/plugins/plugin-list-header/index.jsx

```diff
@@ -0,0 +1,181 @@
+		buttons.push( <span key="plugin-list-header__buttons-global-buttons" className="plugin-list-header__mode-buttons">{ rightSideButtons }</span> );
+
+		return buttons;
+	},
+
+	renderCurrentActionDropdown() {
+		let options = [];
+		let actions = [];
+
+		const hasWpcomPlugins = this.props.selected.some( property( 'wpcom' ) );
+		const isJetpackSelected = this.props.plugins.some( plugin => plugin.selected && 'jetpack' === plugin.slug );
+		const needsRemoveButton = !! this.props.selected.length && ! hasWpcomPlugins && this.canUpdatePlugins() && ! isJetpackSelected;
+
+		if ( this.props.isBulkManagementActive ) {
+			options.push( <DropdownItem key="plugin__actions_title" selected={ true } value="Actions">{ this.translate( 'Actions' ) }</DropdownItem> );
+			options.push( <DropdownSeparator key="plugin__actions_separator_1" /> );
+
+			options.push( <DropdownItem key="plugin__actions_activate" disabled={ ! this.props.haveInactiveSelected } onClick={ this.props.activateSelected }>{ this.translate( 'Activate' ) }</DropdownItem> );
+
+			let deactivateAction = isJetpackSelected
+				? <DropdownItem key="plugin__actions_disconnect" disabled={ ! this.props.haveActiveSelected } onClick={ this.props.deactiveAndDisconnectSelected }>{ this.translate( 'Disconnect' ) }</DropdownItem>
+				: <DropdownItem key="plugin__actions_deactivate" disabled={ ! this.props.haveActiveSelected } onClick={ this.props.deactivateSelected }>{ this.translate( 'Deactivate' ) }</DropdownItem>;
+			options.push( deactivateAction );
+
+			if ( this.hasJetpackSelectedSites() ) {
+				options.push( <DropdownSeparator key="plugin__actions_separator_2" /> );
+				options.push( <DropdownItem key="plugin__actions_autoupdate" disabled={ hasWpcomPlugins || ! this.canUpdatePlugins() } onClick={ this.props.setAutoupdateSelected }>{ this.translate( 'Autoupdate' ) }</DropdownItem> );
+				options.push( <DropdownItem key="plugin__actions_disable_autoupdate" disabled={ hasWpcomPlugins || ! this.canUpdatePlugins() } onClick={ this.props.unsetAutoupdateSelected }>{ this.translate( 'Disable Autoupdates' ) }</DropdownItem> );
+
+				options.push( <DropdownSeparator key="plugin__actions_separator_3" /> );
+				options.push( <DropdownItem key="plugin__actions_remove" className="plugin-list-header__actions_remove_item" disabled={ ! needsRemoveButton } onClick={ this.props.removePluginNotice } >{ this.translate( 'Remove' ) }</DropdownItem> );
+			}
+
+			actions.push( <SelectDropdown compact className="plugin-list-header__actions_dropdown" key="plugin-list-header__actions_dropdown" selectedText="Actions">{ options }</SelectDropdown> );
+		}
+		return actions;
+	},
+
+	render() {
+		const sectionClasses = classNames( 'plugin-list-header', { 'is-bulk-editing': this.props.isBulkManagementActive } );
+		return (
+			<SectionHeader label={ this.props.label } className={ sectionClasses }>
+				{
+					! this.props.isBulkManagementActive
```

Comment:
We could write this check as `this.props.isBulkManagementActive && <BulkSelect key="plugin...`